knack/help.py

```diff
@@ -426,25 +426,25 @@ def _print_items(layouts):
 
     @staticmethod
     def _get_choices_defaults_sources_str(p):
-        choice_str = u'  Allowed values: {0}.'.format(', '.join(sorted([str(x) for x in p.choices]))) \
```

Comment:
Okay with removing the numbers in the curly braces, but the leading `u` needs to remain for now.

Reply:
Done!